misc/acrn-config/launch_config/com.py

```diff
@@ -493,7 +496,10 @@ def dm_arg_set(names, sel, dm, vmid, config):
                 launch_cfg_lib.RE_CONSOLE_MAP['virtio-console(hvc0)']), file=config)
 
     if uos_type in ("CLEARLINUX", "ANDROID", "ALIOS"):
-        print("   -s {},virtio-net,$tap \\".format(launch_cfg_lib.virtual_dev_slot("virtio-net")), file=config)
+        if board_name in ("apl-mrb", "apl-up2"):
+            print("   -s {},virtio-net,$tap \\".format(launch_cfg_lib.virtual_dev_slot("virtio-net")), file=config)
+        else:
+            print("   -s {},virtio-net,tap0 \\".format(launch_cfg_lib.virtual_dev_slot("virtio-net")), file=config)
```

Comment:
why this?

Reply:
i followed the sample board for this on temporary. i will update it on next version